lib/internal/modules/esm/loader.js

```diff
@@ -108,6 +128,7 @@ function nextHookFactory(chain, meta, validate) {
     url: hookFilePath,
   } = chain[meta.hookIndex];
 
+  // ex 'nextResolve'
   const nextHookName = `next${
     StringPrototypeToUpperCase(hookName[0]) +
     StringPrototypeSlice(hookName, 1)
```

Comment:
And what about `nextPreloadCode`? Not applicable because it doesn’t chain through this?

@aduh95 is there a `camelCase` utility function hiding in the codebase somewhere?